### DISCUSSION TITLE
Danish version

### DISCUSSION THREAD

**Discussion by themortenbay** [DivinumOfficium/divinum-officium#2731] [Q&A] [answered]
Dear community

I've been praying the traditional breviary (in different versions) for a couple of years now and I'm considering creating a Danish translation of the 1960 rubrics version of Divinum Officium. It's my mother tongue and I have access to a bible translation without copyright issues, so the biblical texts shouldn't prove too complicated.
I'm however completely new to github. But I have sniffed around a couple of minutes and I did find the file repositories. I figure it's all just about forking the files and getting to it. However I don't want to complicate the process unnecessarily. Would I be able to fork the entire "English" folder of the horas, translate the files at my leasure and just change the folder name to Danish at the end?

Thanks a bunch in advance.
/Morten

**Comment by mbab** [comment, marked as answer]
Welcome to project!
You can fork only whole project.
If You plan to work through Github web interface, then create 'Danish' directory  (should be dansk or danske perhaps) in `web/www/horas` of your fork. Then create any subdirectory (I suggest start with `psalms1` and `Psalterium`) and fulfill it with translated files.
(Working on Your computer should go in similar way.)

**Comment by fiapps** [comment]
Just to elaborate on mbab's answer: For the name of your translation, you could also use `Dansk`: you'll notice that translation names are usually in the language of the translation.

You don't need to copy files from the `English` directory into `Dansk` or `Danish`, and should not do so. When file or file section is missing, the program automatically uses English. This means you can translate one file at a time, or even one section of a file. The Psalms (which don't have sections) are a good place to start, because you can just copy and paste from your translation without having to understand much about how the program works. Another place many start is with commonly used texts, like `Prayers.txt` or the ordinarium.
